samza-sql/src/main/java/org/apache/samza/sql/translator/TranslatorConstants.java

```diff
@@ -20,10 +20,10 @@
 package org.apache.samza.sql.translator;
 
 public class TranslatorConstants {
-  public static final String PROCESSING_TIME_NAME = "processingTimeMs";
-  public static final String TOTAL_LATENCY_NAME = "totalLatencyMs";
-  public static final String QUERY_LATENCY_NAME = "queryLatencyMs";
-  public static final String QUEUEING_LATENCY_NAME = "queueingLatencyMs";
+  public static final String PROCESSING_TIME_NAME = "processingTimeMicros";
```

Comment:
Why is this micros? Same for the PR title.

If this timer is indeed in micros, I'd recommend making it nanos instead for consistency. Timers are usually assumed to be millis or nanos.

Reply:
thanks for the catch. Done